### DISCUSSION TITLE
Get Coordinates for shapes

### DISCUSSION THREAD

**Discussion by salmanedhi** [img-mapper/img-mapper#24] [Q&A] [answered]
Hi, can anyone help me that how I can get the coordinates for new shapes I want to create?

**Comment by NishargShah** [comment, marked as answer]
Poly supports most of the shapes

**Reply by salmanedhi** [reply]
that I am aware of but how to get the coordinates for those shapes? I can't seem to find a way to find coordinates for the polygon

**Reply by salmanedhi** [reply]
> that I am aware of but how to get the coordinates for those shapes? I can't seem to find a way to find coordinates for the polygon

to clear my point. I have this new image and i want to find out all the coordinates for the different shapes I have in that image. Where do I get the coordinates from?

**Reply by NishargShah** [reply]
Its array of longitude, latitude. I think this https://www.keene.edu/campus/maps/tool/ website can help you to understand that how can you make the coordinates.

**Reply by NishargShah** [reply]
Ok, please read first, how the poly map is working https://www.w3schools.com/html/html_images_imagemap.asp then go to this https://www.image-map.net website and generate coordinates.

I hope this is the answer you want.

**Reply by salmanedhi** [reply]
yes, perfect answer! just exactly what I was looking for. Thank you!
